sylvia/tests/dispatching.rs

```diff
@@ -108,6 +128,7 @@ mod contract {
     pub struct Contract {
         pub(crate) execs: RefCell<u64>,
         pub(crate) queries: RefCell<u64>,
+        pub(crate) sudos: RefCell<u64>,
```

Comment:
I didn't notice it before, but it would be much better if those entities would be contract state, and then you can always raw-query them in the test. Could we fix all 4 of those (making data a map) in this PR?

Reply:
Changed all but `queries` as we don't have access to mutable storage query.